src/api/SetTokenAPI.ts

```diff
@@ -152,10 +152,71 @@ export class SetTokenAPI {
     return quantity.mod(naturalUnit).eq(ZERO);
   }
 
+  /**
+   * Calculates the units of components of a Set transferred given an issue or redeem quantity.
```

Comment:
Omit periods in descriptions: https://docs.setprotocol.com/api#getallowanceasync

Reply:
I still don't understand the `Transferred` terminology. Why not: `Calculate the amounts of each component to use in an issue or receive in a redeem of a Set for a given quantity`.